pylint/checkers/match_statements_checker.py

```diff
@@ -46,6 +63,19 @@ class MatchStatementChecker(BaseChecker):
             "Emitted when there is more than one sub-pattern for a specific "
             "attribute in a class pattern.",
         ),
+        "R1905": (
+            "Use '%s' instead",
```

Comment:
```suggestion
            "Use '%s() as %s' instead",
```
One less string formatting and better looking 'base message' in the documentation that way.